packages/ember-views/lib/views/bound_component_view.js

```diff
@@ -0,0 +1,50 @@
+/**
+@module ember
+@submodule ember-views
+*/
+
+import { _Metamorph } from "ember-views/views/metamorph_view";
+import { read, chain, subscribe, unsubscribe } from "ember-metal/streams/utils";
+import { readComponentFactory } from "ember-views/streams/utils";
+import mergeViewBindings from "ember-htmlbars/system/merge-view-bindings";
+import EmberError from "ember-metal/error";
+
+export default Ember.ContainerView.extend(_Metamorph, {
```

Comment:
@lukemelia - `Ember.ContainerView` does much more than what we need (and I think that `OutletView` should likely be updated as well). For example `ContainerView` calls `defineProperty` on itself during `init` (which is generally known to be a perf issue).

I'm not unhappy with your current implementation, I just think it could be better/simpler. I'm happy to wait for the real view experts (obviously not me) to chime in...